terraform/es.tf

```diff
@@ -0,0 +1,35 @@
+resource "aws_elasticsearch_domain" "monitoring-framework" {
+  domain_name           = "tg-${var.environment}-es"
+  elasticsearch_version = "2.3"
+
+  cluster_config {
+    instance_type            = "m4.large.elasticsearch"
```

Comment:
@elaygl Please change to `t2.small.elasticsearch`. our AWS bill will be happier that way

Reply:
done